lib/mongo_storage.js

```diff
@@ -0,0 +1,49 @@
+var db = require('monk');
+
+module.exports = function(path) {
```

Comment:
Nice PR, @lambtron. In order to provide a consistent interface, this should be ```function(options)``` where options is an object with a path property.